packages/venia-styleguide/src/pages/typography/sections/FontSizes/index.md

```diff
@@ -0,0 +1,10 @@
+import Columns from "../../../../components/Columns"
+import FontSize from "./FontSize"
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce quis euismod nisi. Morbi metus mauris, volutpat ac aliquet eget, laoreet vel tortor. Pellentesque commodo tellus nibh, vitae varius lectus pharetra in. Aliquam quis nisi ligula. Proin sit amet mauris ac lacus efficitur varius eget in urna. Ut sagittis feugiat ex et dictum. Nam ut tempor urna, at dapibus erat. Aenean ac dui a tellus venenatis accumsan.
+
+***
+
+<Columns>
+  <FontSize />
```

Comment:
Maybe it's the columns component but it looks like it makes two columns differing by background.